docs/docs/getting-started/web.mdx

```diff
@@ -133,14 +203,21 @@ import { WithSkiaWeb } from "@shopify/react-native-skia/lib/module/web";
 export default function App() {
   return (
     <WithSkiaWeb
-      getComponent={() => import("./MySkiaComponent")}
+      getComponent={() => import("./MySkiaComponent")} // use require(...) if you're using the metro bundler
```

Comment:
I didn't need to use require() with the metro bundler.

Reply:
It works well in dev mode, but fails when you build for prod. #sneaky  👹

You can do that by running `npx expo export --platform web` and serve with `npx serve dist`. 
👉 See [#2017](https://github.com/Shopify/react-native-skia/pull/2017#issuecomment-1880668780)

Reply:
I am not experiencing this issue